modules/nested-elements/assets/js/editor/nested-repeater/controls/repeater.js

```diff
@@ -28,7 +28,7 @@ export default class Repeater extends elementor.modules.controls.Repeater {
 			container: this.options.container,
 			name: this.model.get( 'name' ),
 			index: childView._index,
-			renderAfterInsert: false,
+			renderAfterInsert: childView.renderAfterInsert,
```

Comment:
@mykytamurzin I had a look at this again. I see the `default` for `renderAfterInsert` is `true`, so actually you can just delete this line.